sriov/common/utils.py

```diff
@@ -533,8 +547,8 @@ def execute_and_assert(ssh_obj, cmds, exit_code, timeout=0):
     return outs, errs
 
 
-def execute_until_timeout(ssh_obj, cmd, timeout=10):
-    """Execute cmd and check for 0 exit code until timeout
+def execute_until_timeout(ssh_obj: ShellHandler, cmd: str, timeout: int = 10) -> bool:
+    """ Execute cmd and check for 0 exit code until timeout
```

Comment:
@dkosteck @akiselev1 I thought the standard google style has no space here, and this space was removed earlier to pass the flake check, right?

Reply:
ACK. Just to be clear, here is an example of the google docstring,
https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html